runtime/coreclr-bridge.m

```diff
@@ -244,8 +244,6 @@
 void
 xamarin_install_nsautoreleasepool_hooks ()
```

Comment:
Why is this empty?

Reply:
@dalexsoto Because nothing needs to happen where - there's another version of this method for MonoVM:

https://github.com/xamarin/xamarin-macios/blob/5a0600491c05e34cdc40c765ea4a18c5debab484/runtime/monovm-bridge.m#L283-L290

I've added a comment to explain this.